package.json

```diff
@@ -88,6 +88,7 @@
         "@typescript-eslint/no-var-requires": "off",
         "react/no-find-dom-node": "warn",
         "react/no-unescaped-entities": "off",
+        "react/jsx-no-target-blank": "warn",
```

Comment:
Instead of this, let's silence the warning in the one case below only.

Reply:
@domoritz Do you mean to change it to 'off'?

Reply:
No, use `// eslint-disable-next-line react/jsx-no-target-blank`